modules/virt-disable-individual-bootsource-update.adoc

```diff
@@ -5,13 +5,17 @@
 
 :_content-type: PROCEDURE
 [id="virt-disable-individual-bootsource-update_{context}"]
-= Disabling an automatic update for a custom boot source
+= Disabling an automatic update for a single system-defined or user-defined boot source
 
-You can reduce the number of logs on disconnected environments or reduce resource usage by disabling the automatic imports and updates for an individual custom boot source.
+System-defined boot sources are not listed by default in the `spec.dataImportCronTemplates` of the `HyperConverged` custom resource (CR).
+In order to disable automatic imports and updates for a system-defined boot source, you add the boot source to the `dataImportCronTemplates:` field in the custom resource list and edit the `dataimportcrontemplate.kubevirt.io/enable` annotation.
+
+You can disable automatic imports and updates for a single user-defined boot source and for a single system-defined boot source.
 
 .Procedure
 
-* Edit the `HyperConverged` custom resource (CR) and disable updates for a custom boot source by setting the `dataimportcrontemplate.kubevirt.io/enable` annotation in the CR to `false`.
+* To disable automatic imports and updates for a single user-defined boot source, remove the boot source from the `dataImportCronTemplates` field in the custom resource list.
+* To disable automatic imports and updates for a single system-defined boot source, edit the `HyperConverged` CR and disable automatic imports and updates, setting the `dataimportcrontemplate.kubevirt.io/enable` annotation in the CR to `false`.
```

Comment:
This line is a bit misleading. It is giving me the impression that we are talking about hco.metadata.annotation, instead of metadata.annotation for the system-defined bootsource under HCO CR spec.dataImportCronTemplates. Can this be elaborated a bit?

Reply:
Yes, clarified as:

* To disable automatic imports and updates for a single system-defined boot source, edit the `HyperConverged` CR and disable automatic imports and updates, setting the `spec.dataImportCronTemplates.dataimportcrontemplate.kubevirt.io/enable` annotation in the CR to `false`.